### PR TITLE
adding pointer-events:none to bb-context-menu li disabled

### PR DESCRIPTION
**Pull request by josh-channin**
#479

### CODE REVIEW

**Comment by blackbaud-sky-savage** [comment]
**Tests passed.** Automated cross-browser testing via BrowserStack and Travis CI shows that the JavaScript changes in this pull request are: **CONFIRMED**

Commit: 752bd4d66a808cb5b9d0aa74a10c17bb137ef7cb
Build details: https://travis-ci.org/blackbaud-sky-savage/skyux/builds/124803193

(_Please note that this is a [fully automated](https://github.com/twbs/savage) comment._)

File: scss/_dropdowns.scss [hunk 1]
```diff
@@ -120,6 +120,10 @@
 }
 
 .bb-context-menu .dropdown-menu {
+  
+    li:disabled {
```
Comment (Blackbaud-PatrickOFriel) [inline scss/_dropdowns.scss]: One change here, I think it should be working off of the disabled class so the selector should be `li.disabled` instead

**Comment by blackbaud-sky-savage** [comment]
**Tests failed.** Automated cross-browser testing via BrowserStack and Travis CI shows that the JavaScript changes in this pull request are: **BUSTED**

Commit: 1e3ded579b14c5bb2989a8629e747337827aa2a5
Build details: https://travis-ci.org/blackbaud-sky-savage/skyux/builds/125118915

(_Please note that this is a [fully automated](https://github.com/twbs/savage) comment._)

**Comment by josh-channin** [comment]
@blackbaud-sky-savage retry

**Comment by Blackbaud-PatrickOFriel** [comment]
@blackbaud-sky-savage retry

**Comment by blackbaud-sky-savage** [comment]
**Tests failed.** Automated cross-browser testing via BrowserStack and Travis CI shows that the JavaScript changes in this pull request are: **BUSTED**

Commit: 1e3ded579b14c5bb2989a8629e747337827aa2a5
Build details: https://travis-ci.org/blackbaud-sky-savage/skyux/builds/125130145

(_Please note that this is a [fully automated](https://github.com/twbs/savage) comment._)

**Comment by blackbaud-sky-savage** [comment]
**Tests passed.** Automated cross-browser testing via BrowserStack and Travis CI shows that the JavaScript changes in this pull request are: **CONFIRMED**

Commit: 864c8f5328017a3dc2e2f58d89a19947fb87ef9a
Build details: https://travis-ci.org/blackbaud-sky-savage/skyux/builds/125131502

(_Please note that this is a [fully automated](https://github.com/twbs/savage) comment._)

**Comment by Blackbaud-PatrickOFriel** [comment]
Approved

[Approved with PullApprove](https://pullapprove.com/blackbaud/skyux/pull-request/516/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=blackbaud/skyux)

**Pull request merged**